package.json

```diff
@@ -34,23 +34,23 @@
     "eslint": "8.32.0",
     "eslint-config-prettier": "6.15.0",
     "husky": "1.3.1",
-    "jest": "28.1.3",
+    "jest": "^29.7.0",
     "jest-junit": "14.0.1",
     "jsdom": "^16.5.0",
     "lerna": "5.5.4",
     "markdown-link-check": "^3.9.3",
     "ncp": "^2.0.0",
+    "ovsx": "latest",
     "prettier": "1.19.1",
     "shelljs": "0.8.5",
     "shx": "0.3.4",
     "snyk": "^1.1017.0",
-    "ts-jest": "28.0.8",
+    "ts-jest": "^29.1.1",
     "ts-loader": "^9.3.0",
     "ts-node": "10.9.1",
     "tslint": "5.5.0",
-    "typescript": "^4.7.4",
-    "vsce": "2.15.0",
-    "ovsx": "latest"
+    "typescript": "^5.2.2",
```

Comment:
The typescript 5 change was required in order to use the node18 ts config build.  Otherwise it couldn't find the ES2023 value defaulted in the  node18 config file. "@tsconfig/node18/tsconfig.json",